adapters/InMobi/InMobiAdapter/GADMAdapterInMobiUtils.m

```diff
@@ -193,3 +193,8 @@ void GADMAdapterInMobiSetIsAgeRestricted(NSNumber *_Nullable isRestricted) {
 
   return requestParameters;
 }
+
+void GADMAdapterInMobiSetPrivacyCompliance(void) {
```

Comment:
Since this is U.S. Privacy specific, we can rename it to: `GADMAdapterInMobiSetUSPrivacyCompliance`.